qutebrowser/browser/commands.py

```diff
@@ -451,9 +451,10 @@ def tab_take(self, index, keep=False):
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
     @cmdutils.argument('win_id', completion=miscmodels.window)
+    @cmdutils.argument('private', flag='p')
     @cmdutils.argument('count', value=cmdutils.Value.count)
     def tab_give(self, win_id: int = None, keep: bool = False,
-                 count: int = None) -> None:
+                 count: int = None, private=False) -> None:
         """Give the current tab to a new or existing window if win_id given.
 
         If no win_id is given, the tab will get detached into a new window.
```

Comment:
You'll also need to add a description for the `private` argument in this docstring (below)